Libs/MRML/DisplayableManager/vtkMRMLViewInteractorStyle.cxx

```diff
@@ -71,31 +76,19 @@ void vtkMRMLViewInteractorStyle::PrintSelf(ostream& os, vtkIndent indent)
 //----------------------------------------------------------------------------
 void vtkMRMLViewInteractorStyle::OnKeyPress()
 {
-  if (this->DelegateInteractionEventToDisplayableManagers(vtkCommand::KeyPressEvent))
-    {
-    return;
-    }
-  this->Superclass::OnKeyPress();
+  this->GetInteractorStyle()->OnKeyPress();
```

Comment:
If all we do is calling the Superclass implementation then it is better to remove the method completely.

Reply:
This is intended to keep the same behavior as we had before, I will create a follow-up pull request and/or issue to move forward with simplification